### ISSUE TITLE
[Enhancement] Change Damage Type

### ISSUE THREAD

**Issue by doxa97** [magemonkeystudio/fabled#268]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

Regarding damage, only the type of damage is ENTITY ATTACK. It would be nice if this type could be changed to another type! When linking with the plug-in I'm making, I tried using the POTION effect because of this part, but it was recognized that this effect was given by the server itself, not the caster's effect on the target. If the damage type can be specified as custom, I think it will be possible to determine the exact amount of damage done by varying the damage type for each skill. For example, when giving MAGIC damage, the damage is amplified by 1 of the stat, and when giving DROWN damage, it is given by 0.5!

I'm Korean, so I'm sorry that I didn't translate it properly through a translator, and if you reflect it, I think I can make a really good RPG!

+ Or maybe the feature I'm talking about is in the paid version?

**Comment by lordsrubka5** [comment]
Yes, all damage is ENTITY ATTACK however in the damage mechanic you can use something called **classifier** to give an ID to some damages which later you can modify with attributes or make it scale in a different way or to use them in skill damage / took skill damage conditions. I use these to make a difference between normal skill damages and damage over time like poison, bleed and burn.
